features/admin.users.v1/components/AccountStatusFilterDropdown.scss

```diff
@@ -0,0 +1,62 @@
+/**
+ * Copyright (c) 2025, WSO2 LLC. (https://www.wso2.com).
+ *
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+$primary-color: #ff7300;
+$badge-text-color: white;
+$badge-dimension: 20px;
+$select-background-color: #fafafa;
+$select-height: 40px;
+
+.account-status-filter-dropdown {
+    margin-right: 5px;
+    min-width: 200px;
+
+    .dropdown-label-container {
+        align-items: center;
+        display: flex;
+        justify-content: space-between;
+        width: 100%;
+
+        .status-count-badge {
+            margin-right: 5px;
+
+            .MuiBadge-badge {
+                background-color: $primary-color;
+                border-radius: 50%;
```

Comment:
Check whether this required

Reply:
Resolved via https://github.com/wso2/identity-apps/pull/8171/commits/a35f3f308db99f0f624552fce3446dbdefa8b471